src/icons/svg/vue.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="1536" height="1792" viewBox="0 0 1536 1792" xmlns="http://www.w3.org/2000/svg">
+    <path d="m 0,341.7508 19.165,33.24858 v 0 0.013 L 639.9498,1450.2492 1260.7469,374.99971 v 0 L 1280,341.7508 v 0 H 1024.0335 787.75299 749.62765 L 640.01369,531.06776 530.38539,341.7508 H 492.24701 255.7761 Z m 274.979,33.24858 H 511.47528 L 639.98761,597.7177 768.55014,374.99938 H 1004.8046 L 639.98761,1006.8369 Z" />
```

Comment:
I still see the white-space removal diff when I build this locally. Maybe you do not. So, I will leave it at that.

Reply:
I removed it